docs/AIRGAP.md

```diff
@@ -0,0 +1,7 @@
+# Airgap Flavor
+
+This package can be deployed fully airgapped by using the `airgap` flavor. All of the images required for the `k3d` cluster to spin up are loaded into `docker` using `docker load` before the cluster is created. Then all of the images required for the `uds-dev-stack` are loaded into the `k3d` cluster using `k3d image load`.
```

Comment:
Potentially related to the comment on `K3D_HELPER_IMAGE_TAG` - should we list a limitation on k3d CLI version required for this? Either for just the airgap variant or maybe update our prerequisites - https://github.com/defenseunicorns/uds-k3d?tab=readme-ov-file#prerequisites - as needed?

Reply:
_should_ work regardless of local k3d version because we're overriding the helper image version and specifying the airgap images.